README.md

```diff
@@ -34,8 +34,7 @@ An extremely fast Python linter and code formatter, written in Rust.
 - 🔧 Fix support, for automatic error correction (e.g., automatically remove unused imports)
 - 📏 Over [800 built-in rules](https://docs.astral.sh/ruff/rules/), with native re-implementations
     of popular Flake8 plugins, like flake8-bugbear
-- ⌨️ First-party [editor integrations](https://docs.astral.sh/ruff/integrations/) for
-    [VS Code](https://github.com/astral-sh/ruff-vscode) and [more](https://docs.astral.sh/ruff/editors/setup)
+- ⌨️ First-party integrations for [VS Code](https://github.com/astral-sh/ruff-vscode) or [other editors](https://docs.astral.sh/ruff/editors) like [Neovim](https://docs.astral.sh/ruff/editors/setup/#neovim), and [more development automation and environment management tools]([https://docs.astral.sh/ruff/editors/setup](https://docs.astral.sh/ruff/integrations)
```

Comment:
I don't think it's correct to call these first-party integration:

* neovim: The neovim plugin isn't maintained by us
* other editors: The same is true for other editors, e.g. the zed extension is built by someone else
* We also don't provide first-party support for all integrations listed under integrations.